pkg/webhook/sidecar_spec.go

```diff
@@ -28,16 +28,18 @@ const (
 	SidecarContainerName            = "gke-gcsfuse-sidecar"
 	SidecarContainerVolumeName      = "gke-gcsfuse-tmp"
 	SidecarContainerVolumeMountPath = "/gcsfuse-tmp"
+	CacheVolumeName                 = "cache-volume"
```

Comment:
Can we rename the volume name to `gke-gcsfuse-cache-volume`, and change the mount path to `/gcsfuse-cache`? This is because we have to document this reserved volume so that users won't use them in other containers. I think "cache-volume" is very general and users may want to use this volume name for other containers.

Reply:
Resolving as discussed offline. This feature no longer necessitates driver changes.